lib/vsc/install/shared_setup.py

```diff
@@ -1353,9 +1358,6 @@ def action_target(package, *args, **kwargs):
     """
     This main is the setup.py for vsc-install
     """
-    install_requires = [
```

Comment:
there's a vsc environment variable in vsc-install to control the renaming of the dependencies, but clusterbuildrpm does the actually renaming of the rpm using rpmrebuild. i have not looked into patching bdist_rpm to produce the "correct" python- named rpms itself.